jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/SamProjectWizard.kt

```diff
@@ -147,21 +149,15 @@ abstract class SamProjectTemplate {
             hasException = true
             throw e
         } finally {
-            TelemetryService.getInstance().record(project) {
-                datum("SAM.Init") {
-                    metadata("name", getName())
-                    metadata("runtime", runtime.name)
-                    metadata("samVersion", SamCommon.getVersionString())
-                    metadata("hasException", hasException)
-                    metadata("hasSchema", schemaParameters != null)
-                    metadata("template", this.javaClass.simpleName)
-                    schemaParameters?.let {
-                        if (it.schema.registryName == AWS_EVENTS_REGISTRY) {
-                            metadata("awsSchema", it.schema.name)
-                        }
-                    }
-                }
-            }
+            SamTelemetry.init(
+                project,
+                name = getName(),
+                result = if (hasException) Result.CANCELLED else Result.FAILED,
```

Comment:
this has no cancel ability. This line is wrong

Reply:
Yep this was just wrong, fixed